docs/Deprecating-Disabling-and-Removing-Formulae.md

```diff
@@ -18,6 +18,11 @@ A formula should be deprecated to indicate to users that the formula should not
 
 The most common reasons for deprecation are when the upstream project is deprecated, unmaintained, or archived.
 
+Formulae with dependents should **not** be deprecated unless the formula:
+
+- does not build on any of our supported platforms; or,
+- has outstanding CVEs.
```

Comment:
Or if all dependents are deprecated.

Reply:
Oh, yes. Had intended to include that.

Reply:
Done. We should probably have a tap syntax check for formulae with dependents that are deprecated.

Reply:
> Done. We should probably have a tap syntax check for formulae with dependents that are deprecated.

Agreed 👍🏻. Seems sensible for `brew audit` to handle.